bentoml/sklearn.py

```diff
@@ -125,7 +125,7 @@ def __init__(
         batch_options: t.Optional[t.Dict[str, t.Any]],
         model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
     ):
-        super().__init__(str(tag), resource_quota, batch_options)
+        super().__init__(f"{tag}-{function_name}", resource_quota, batch_options)
```

Comment:
is this change necessary? I think it's ok to just use the model name so everything is consistent. Doing this will be hard for some other frameworks, xgboost's load_runner for example takes a whole dictionary, and you can't easily get a readable and unique str to represent the runner.

A slightly nicer way to do it is:

```suggestion
        super().__init__(f"{tag}-{id(self)}", resource_quota, batch_options)
```

Reply:
```suggestion
        super().__init__(str(tag), resource_quota, batch_options)
```